### PR TITLE
Improve WImage

### PR DESCRIPTION
**Pull request by 7alip**

### CODE REVIEW

File: libs/ui/src/components/WImage/WImage.tsx [hunk 1]
```diff
@@ -1,91 +1,74 @@
...
+        <rect id="r" width="${width}" height="${height}" fill="url(#g)" />
+        <animate xlink:href="#r" attributeName="x" from="-${width}" to="${width}" dur="1s" repeatCount="indefinite"  />
+      </svg>`
 
 export type WImageProps = {
   ratio?: number | 'twitter'
   format?: FileFormatsType
   image: UploadFile | string
   alt: string
   isExternal?: boolean
```
Comment (MustafaTRKYLMZ) [inline libs/ui/src/components/WImage/WImage.tsx:29]: Should has WImage hasZoom props? I think, it should has. What do you think?

File: libs/ui/src/components/WImage/WImage.tsx [hunk 2]
```diff
@@ -1,91 +1,74 @@
...
-      'objectFit',
-    ].includes(prop),
-})
-
 // TODO: add loader
 export const WImage: FC<WImageProps> = ({
   image,
   format,
   alt,
   ratio,
```
Comment (MustafaTRKYLMZ) [inline libs/ui/src/components/WImage/WImage.tsx:38]: hasZoom props?

**Review by MustafaTRKYLMZ — APPROVED** [review]
Everytthing looks good except for hashZoom prop. Maybe it is because of WImage-zoom pr.

File: libs/ui/src/components/WImage/WImage.tsx (hunk 2 above)
Comment (7alip) [reply libs/ui/src/components/WImage/WImage.tsx:38]: hasZoom was in your PR. It will be included once I merge dev into this branch

**Comment by 7alip** [comment]
@MustafaTRKYLMZ I merged dev into fix/image component. Could you test WImage and other components that we use WImage. If everythink looks fine, you can merge it

**Pull request merged**